zsh-history-substring-search.zsh

```diff
@@ -48,6 +48,7 @@ typeset -g HISTORY_SUBSTRING_SEARCH_HIGHLIGHT_NOT_FOUND='bg=red,fg=white,bold'
 typeset -g HISTORY_SUBSTRING_SEARCH_GLOBBING_FLAGS='i'
 typeset -g HISTORY_SUBSTRING_SEARCH_ENSURE_UNIQUE=''
 typeset -g HISTORY_SUBSTRING_SEARCH_FUZZY=''
+typeset -g HISTORY_SUBSTRING_SEARCH_PREFIX=''
```

Comment:
Should we use `true`/`false` values for boolean feature flags rather than non/empty strings?

Reply:
It seems that all the other flags are using the non/empty pattern. I don't think it's good to introduce inconsistency.

Reply:
Good point, thanks for checking this.